inference-engine/src/vpu/graph_transformer/src/middleend/passes/allocate_resources.cpp

```diff
@@ -224,6 +224,20 @@ AllocationResult runAllocator(const Model& model, EnableShapeAllocation enableSh
                 allocateShape(tempBuffer);
             }
         }
+
+        // Allocate shape for unused inputs
+        DataVector unusedInputs;
+        const auto& dataObjects = model->datas();
+        std::copy_if(dataObjects.begin(), dataObjects.end(), std::back_inserter(unusedInputs), [](const Data& data) {
+            return data->usage() == DataUsage::Input && !data->isConsumed();
+        });
+        std::sort(unusedInputs.begin(), unusedInputs.end(), [](const Data& lhs, const Data& rhs) { return lhs->name() < rhs->name(); });
```

Comment:
Please mention in the comment why you need to sort it

Reply:
Done.